packages/test-exclude/test/test-exclude.js

```diff
@@ -53,6 +53,28 @@ describe('testExclude', () => {
             .should.equal(false);
     });
 
+    it('ignores fixture directories and files', () => {
+        exclude()
+            .shouldInstrument('./fixture/index.js')
+            .should.equal(false);
+
+        exclude()
+            .shouldInstrument('./fixtures/index.js')
+            .should.equal(false);
+
+        exclude()
+            .shouldInstrument('./fixture-some-suffix/index.js')
+            .should.equal(false);
+
+        exclude()
+            .shouldInstrument('./nested/fixtures/index.js')
+            .should.equal(false);
+
+        exclude()
+            .shouldInstrument('./fixtures-file.txt')
```

Comment:
I think the default extension list would be excluding this, not the `fixture` name

Reply:
test-exclude doesn't actually have a default extension list, that's only in nyc.  Maybe this should change before the final semver-major release, move the default extension list from nyc to here?

In any case I'm leaning towards agreeing with @JaKXz that the tests here should just test against `.js` files.